src/animation/interpolants/SmoothInterpolant.js

```diff
@@ -0,0 +1,103 @@
+/**
+ *
+ * Cubic hermite spline interpolant.
+ *
+ *
+ * @author tschw
+ */
+
+THREE.SmoothInterpolant = function( times, values, stride, result ) {
+
+	THREE.Interpolant.call( this, times, values, stride, result );
+
+};
+
+Object.assign( THREE.SmoothInterpolant.prototype, THREE.Interpolant.prototype, {
+
+	constructor: THREE.SmoothInterpolant,
+
+	DefaultParameters: {
```

Comment:
Interesting to have the velocities. Usually I see people control this via just having more keys, rather than additional parameters.

I do know of Interpolators with a lot of parameters, but they are generally not used in real-time contexts -- like the bias/continuity/tensition smooth interpolator,